Dockerfile.ubi8

```diff
@@ -0,0 +1,42 @@
+FROM golang:1.18.1 AS build
+WORKDIR /go/src/github.com/aquasecurity/kube-bench/
+COPY makefile makefile
+COPY go.mod go.sum ./
+COPY main.go .
+COPY check/ check/
+COPY cmd/ cmd/
+COPY internal/ internal/
+ARG KUBEBENCH_VERSION
+RUN make build && cp kube-bench /go/bin/kube-bench
+
+
+# ubi8-minimal base image for build with ubi standards
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5 as run
+
+RUN RUN microdnf install yum \
+  && yum -y update-minimal --security --sec-severity=Moderate --sec-severity=Important --sec-severity=Critical \
+  && yum clean all \
+  && yum install glibc \
+  && yum update -y glibc \
```

Comment:
Could please include the `jq` package, the `procps` package and override the `PATH` env?:

https://github.com/aquasecurity/kube-bench/blob/d190cbcaf66d2681c8782081f51c3e9afaef5eb4/Dockerfile#L14-L16

https://github.com/aquasecurity/kube-bench/blob/d190cbcaf66d2681c8782081f51c3e9afaef5eb4/Dockerfile#L31-L33

Reply:
Done